### DISCUSSION TITLE
차단하는 기능에 저장 가능한 최대치가 있나요?

### DISCUSSION THREAD

**Discussion by RoTuSil** [green1052/DCRefresher-Reborn#14] [Q&A] [answered]
어쩌다보니 차단을 좀 많이 하게 됬는데..
일정 수 이상 차단을 하고 나니 더 차단이 안됩니다.
차단을 해도 그 당시에는 글이 없어지는 애니메이션이 나오면서 확장 프로그램 내에도 차단목록에 추가가 된 게 보이는데
새로고침하니 그 글도 다시 보이고 차단 목록에서도 사라지네요.. 만약 최대치가 있는거라면 어떻게 늘리는 방법이 없을까요?

**Comment by green1052** [comment, marked as answer]
얼마나 차단하셨는지 알 수 있을까요?

**Reply by RoTuSil** [reply]
현재 id 138개, ip 110개 차단 돼 있습니다. 저도 이렇게 많은줄은 몰랐네요..
id와 ip 모두 추가적인 차단이 안되는 상태입니다.
id와 ip 외 다른 차단 키워드는 설정되있지 않습니다.

**Reply by RoTuSil** [reply]
차단 시도시 콘솔에 이런 오류가 뜨네요.
![image](https://user-images.githubusercontent.com/64220019/213764775-e054173e-1300-43cc-b772-425329d517d7.png)

**Reply by green1052** [reply]
[docs](https://developer.chrome.com/docs/extensions/reference/storage/)

```
QUOTA_BYTES_PER_ITEM
8192

The maximum size (in bytes) of each individual item in sync storage, as measured by the JSON stringification of its value plus its key length. Updates containing items larger than this limit will fail immediately and set [runtime.lastError](https://developer.chrome.com/docs/extensions/reference/runtime/#property-lastError).
```

차단된 갯수가 너무 많아서 그런거네요
데이터 크기 제한이 있는 sync 저장소 대신 
local 저장소에 데이터를 저장하는 옵션을 추가하겠습니다.

**Reply by RoTuSil** [reply]
늦은 시간에도 감사드립니다. 항상 잘 쓰고 있습니다!

**Comment by green1052** [comment]
[3.8.0](https://github.com/green1052/DCRefresher-Reborn/releases/tag/3.8.0)에서 반영됐습니다.
